src/ProjectConfiguration.ts

```diff
@@ -304,7 +304,7 @@ export function normalizeLocales(locales: LocalesConfig) {
  */
 export function normalizeProjectConfig(
   config: any,
-  defaults?: Partial<AppProjectConfiguration & ClockProjectConfiguration>,
+  defaults?: Partial<AppProjectConfiguration | ClockProjectConfiguration>,
```

Comment:
TS got smarter, it realised this could never actually happen because there was no overlap in the types. 💯